### DISCUSSION TITLE
Update Slicer configs?

### DISCUSSION THREAD

**Discussion by tycannon** [bassamanator/Sovol-SV06-firmware#193] [General]
Hi, I noticed that the slicer profiles you based yours on have been updated more recently than yours. Do you incorporate those changes? Or once you based your profiles off them, they are more accurately tuned for your needs? Thanks! (https://github.com/mjonuschat/PrusaSlicer-Profiles)

**Comment by bassamanator** [comment]
I pulled in those changes about 2 weeks ago, and created a new PS config bundle that is *currently being tested*. I'm ready to release them now actually, but away from home. Should be out next week.

**Reply by tycannon** [reply]
Excellent, thanks for the update!

**Comment by bassamanator** [comment]
New bundle pushed. Enjoy.
